back/app/models/idea.rb

```diff
@@ -126,6 +126,7 @@ def sanitize_body_multiloc
     )
     self.body_multiloc = service.remove_multiloc_empty_trailing_tags(self.body_multiloc)
     self.body_multiloc = service.linkify_multiloc(self.body_multiloc)
+    self.body_multiloc = service.convert_ampersands_multiloc(self.body_multiloc)
```

Comment:
@kogre Simply removing the sanitization if the Project description preview multiloc does, indeed, seem to work.

The outstanding issue is to prevent the '\&amp;' in the display of the shortened version of the Idea description body text, seen on Idea cards in the Project show view:
<img width="631" alt="ampersands in Idea description" src="https://user-images.githubusercontent.com/3944042/131538666-415a697d-b801-40b5-a689-30f806b2e466.png">

Unfortunately, simply removing the sanitization of the Idea describtion body multiloc does not seem to fix the issue. My naive fix, called in this line (which basically uses `gsub('&amp;', '&')` _does_ seem to work ,however.

I am guessing this 'fix' falls foul of the same drawbacks as for the Project related fix. Any pointers on how to proceed / where to look for clues? Thanks.

Reply:
Great that the first fix works :)

We should be careful with removing sanitization, because it's dangerous in case we are rendering this anywhere as html. We established that it is not the case for the project preview desciprion. But it definitely is the case for the idea body, since it has a rich text editor for the user, so that's a no go.

In fact, we actually want `&amp;` being persisted in the idea body field, because it's the correct way to represent an ampersand in html, the reason why the sanitizer converts it in the first place. So the naive fix is not the way to go either.

I think the best way to deal with this is to do it in the front-end, when we're rendering this card, for this specific case. It's actually an html field, but the FE wants to render a variation without markup (without html) in this specific place. So a similar gsub operation, but then in the FE, should do it.

Alternatively, we could use a library to strip all HTML in the FE, which might also take care of the amp and be more robust all together. This could have the advantage of letting us retain some HTML elements like newlines, if the library offers that as a configuration option, and then still render it as html but with any tags that don't belong there stripped out (e.g. headers, links, bold text...). If we already have a library that does that in the FE, would be a nicer solution I think, if not I'd go with the easy replacement code (I think Sergii already did something like that in his PR) for the idea card. @SaraBCL do you know whether we already have a sanitization lib in the FE somewhere?

Reply:
In the meantime, Iv'e gone with using `.replaceAll('&amp;', '&')` on the `bodyText` in `components/IdeaCard/Compact/index.tsx`

Reply:
I think we don't have a sanitization library in the front-end, would be a good idea actually, and/or expand our localization tools (T, useLocalize, injectLocalize) with the ability to sanitize HTML. This is also leading to think maybe we should be more explicit what's sanitized in the BE and what's not, so what's OK to render as HTML or not, would there be a way to indicate that in the BE documentations ?
I'm ok with the current solution though, if urgent it can go.

Reply:
@kogre In the light of Sara's comments, I suggest we go with this solution for now.  

How about I add the idea of a FE sanitization library as a ticket? Maybe a TEC ticket?

Reply:
Yes, agreed!

Reply:
Created related TEC ticket: [Add FE sanitization library + clarify BE sanitization](https://citizenlab.atlassian.net/browse/TEC-80)